### ISSUE TITLE
Problem with integration

### ISSUE THREAD

**Issue by ivkusto** [matheusmariano/react-native-universal-pedometer#2]
If i understood correctly, you have written pedometer for both platform.
Could you give an example of usage?

**Comment by vitor-mariano** [comment]
Yes, I did specially for the projects of my company, so I hadn't much time to write any docs for this. I'll find some examples to show you some time. :)

**Comment by ivkusto** [comment]
Sorry for disturbing) I tried many others sollutions, and finally i found, that problem was in changing Android device to new model, wich have no step detector/counter)

**Comment by vitor-mariano** [comment]
This plugin is able to deal with android devices without step counter. Maybe we're facing a new issue here. :)

**Comment by Dannnyho** [comment]
So i just move these 2 platform folder in my react project and link up the index.js then it will work?
thanks
